src/posts/blogs/2019/2019-12-04-layer5-landscape-spectrums-revisited/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Layer5: Landscape Spectrums Revisited"
+subtitle: ""
+date: 2019-12-04 18:10:00 +0000
+author: Rafi Ungar
+thumbnail: ./landscape.png
```

Comment:
Whenever an SVG is available, let's choose an SVG over a PNG.  I will provide one that can be used.

Reply:
> Whenever an SVG is available, let's choose an SVG over a PNG.  I will provide one that can be used.

Cool, I will update it.

Reply:
Oh, no. That's not good. 

Here's a [landscape_green.svg](https://github.com/layer5io/layer5/pull/532/files?file-filters%5B%5D=.json&file-filters%5B%5D=.png&file-filters%5B%5D=.svg#diff-0bb95e56e4d2c13658c0ae5e46b16811)

Reply:
I will look into it.

Reply:
Thank you. Given that this is a bigger question than just these couple blog posts, let's create a separate issue to track it so that we can advance your PR toward being merged in the meantime.